release_notes.md

```diff
@@ -1,3 +1,78 @@
+***********************************
+# Release Notes for OpenMDAO 3.17.0
+
+March 21, 2022
+
+OpenMDAO 3.17.0 adds a few significant capabilities, as well as several bugfixes and performance improvements.
+
+The first major capability is the addition of a reports system.  OpenMDAO has provided various feedback through
+standard output, either automatically or at the users request.  The new report capability will allow OpenMDAO to provide
+richer feedback via html and other outputs.  In addition, reports that are useful and cheap to generate, such as the N2
+diagram and the scaling report, will be generated automatically and placed in a reports directory.  The location of
+this reports directory is user-configurable, and options exists for disabling reports when they are unwanted.  Finally,
+the reports system is intended to be extensible, so developers of tools which use OpenMDAO will be able to generate
+their own reports.
+
+OpenMDAO will now also allow aliases for constraints.  This feature will enable users to apply multiple constraints
+to the same variable - such as imposing a equality constraint on one array element and lower/upper bounds on other
+elements.  In the past, users would typically create new "pass-thru" components that would accept different portions
+of these array outputs.  Using the OpenMDAO model hierarchy to "namespace" constraints is also no longer necessary,
+since aliases can be used to give a more human-readable description of the constraint.
+
+We're continuing to add visualization tools. This release features OptionsWidget and CaseViewer.  OptionsWidget provides
+a simple GUI to change options of a model in a Jupyter notebook.  CaseViewer provides a GUI for rapidly plotting
+variables across case iterations from a case record file.  CaseViewer currently only works in a Jupyter notebook,
+but we plan on allowing it to be used as a standalone tool in the future.
```

Comment:
Did you want to say something here about the CaseViewer widget currently only supporting one plot per notebook?